egs/wsj/s5/utils/data/get_reco2dur.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Copyright 2016  Johns Hopkins University (author: Daniel Povey)
+# Apache 2.0
+
+# This script operates on a data directory, such as in data/train/, and adds the
+# reco2dur file if it does not already exist.  The file 'reco2dur' maps from
+# utterance to the duration of the utterance in seconds.  This script works it
```

Comment:
documentation si still wrong: 'utterance to' -> 'recording-id to'; and also the statements about the 'segments' file are wrong.